src/Surfnet/ServiceProviderDashboard/Domain/Entity/Entity/MetaData.php

```diff
@@ -93,7 +93,7 @@ public static function fromApiResponse(array $data)
         Assert::string($nameNl);
 
         if (count(array_unique($acsLocations)) !== count($acsLocations)) {
-            throw new Exception('Double acs locations. Expected unique locations');
+            throw new Exception('Double acs locations. Expected unique locations for entity: ' . $entityId);
```

Comment:
Nitpick hour :warning:  Our go-to standard is to use `sprintf` for string concatenations. Given this is a simple enough onstruction, feel free to leave as-is!